docs/sytest.md

```diff
@@ -10,19 +10,33 @@ passes.
 
 ## Finding out which tests to add
 
-We recommend you run the tests locally by manually setting up SyTest or using a
-SyTest docker image. After running the tests, a script will print the tests you
-need to add to `sytest-whitelist`.
+We recommend you run the tests locally by using the SyTest docker image or
+manually setting up SyTest. After running the tests, a script will print the
+tests you need to add to `sytest-whitelist`.
 
 You should proceed after you see no build problems for dendrite after running:
 
 ```sh
 ./build.sh
 ```
 
+### Using a SyTest Docker image
+
+Ensure you have the latest image for SyTest, then run the tests:
+
+```sh
+docker pull matrixdotorg/sytest-dendrite
+docker run --rm -v /path/to/dendrite/:/src/ matrixdotorg/sytest-dendrite
```

Comment:
Would be good to add a note about also being able to map `/logs` to get the homeserver logging output and the results tap.

Reply:
I think it's common enough to be more than a note.